xarray/tests/test_backends.py

```diff
@@ -5811,7 +5811,7 @@ def _getitem(self, key):
 
     with warnings.catch_warnings(record=True) as w:
         warnings.simplefilter("always")
-        la.vindex[indexer].get_duck_array()
+        la.vindex[indexer.tuple].get_duck_array()
```

Comment:
```suggestion
        la[indexer].get_duck_array()
```

this seems like a good place to check for backcompat warnings in custom backends.

Reply:
i intend to look into this later today